### PR TITLE
[ZH] Add `Account support team`

### PR DESCRIPTION
**Pull request by FlyingFish23**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

announcement message is a new thing, are there any official Chinese name for it yet? i didn't find it and the main page is not translated as well so I just translated the name on my own

### CODE REVIEW

File: wiki/People/Account_support_team/zh.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+# 账号支持团队
+
+*对于管理特定子论坛的团队，请参见[技术支持团队](/wiki/People/Technical_Support_Team)*。
+
+**账号支持团队**是管理用户账号，并接受禁言或账号限制申诉的一个团队。
```
Comment (CloneWith) [inline wiki/People/Account_support_team/zh.md:5]:
```suggestion
**账号支持团队** 是管理用户账号，并接受禁言或账号限制申诉的一个团队。
```
段首的粗体标志末尾最好加一个空格，会导致渲染问题（个人之前经历过）

File: wiki/People/Account_support_team/zh.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+## 联系方式
+
+**只能**通过给[accounts@ppy.sh](mailto:accounts@ppy.sh)，[support@ppy.sh](mailto:support@ppy.sh)，[privacy@ppy.sh](mailto:privacy@ppy.sh)或[tournaments@ppy.sh](mailto:tournaments@ppy.sh)发送邮件来联系账号支持团队。为了保护他们不受到太多无关的关注，团队成员的身份是保密的。
```
Comment (CloneWith) [inline wiki/People/Account_support_team/zh.md:9]:
```suggestion
**只能**通过给[accounts@ppy.sh](mailto:accounts@ppy.sh)，[support@ppy.sh](mailto:support@ppy.sh)，[privacy@ppy.sh](mailto:privacy@ppy.sh)或[tournaments@ppy.sh](mailto:tournaments@ppy.sh)发送邮件来联系账号支持团队。为了保护团队成员免受不需要的关注，他们的身份是保密的。
```

`unwanted`: 不需要，多余的
`无关`: irrelevant
或者这里是意译的？感觉有点偏

File: wiki/People/Account_support_team/zh.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+## 角色与职责
+
+*主条目：[帮助中心](/wiki/Help_centre)*\
+*请参阅：[账号限制](/wiki/Help_centre/Account_restrictions)*
+
+此团队受理你本人无法掌控的账号问题。请记得发送邮件时使用你 osu! 账号所绑定的邮箱，并且详述你的问题。
+
+### [accounts@ppy.sh](mailto:accounts@ppy.sh)
+
+- [对于你账号上限制的申诉](/wiki/Help_centre/Account_restrictions)，包括：
```
Comment (CloneWith) [inline wiki/People/Account_support_team/zh.md:22]:
```suggestion
- [对于你账号所受限制的申诉](/wiki/Help_centre/Account_restrictions)，包括：
```
个人认为“账号受到限制”比较好，用“上”稍显生硬

File: wiki/People/Account_support_team/zh.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  - 禁言；
+  - 个人资料上内容的移除；
+  - 谱面的移除；
+  - 论坛帖和评论的移除。
+- 你[想要承认](/wiki/Reporting_bad_behaviour/Handling_foul_play#我要是违规了怎么办？)的不当行为。
+- [失去了账号所绑定的邮箱](/wiki/Help_centre/Account#sign-in)，或账号被盗。
```
Comment (CloneWith) [inline wiki/People/Account_support_team/zh.md:28]:
```suggestion
- [无法使用账号所绑定的邮箱](/wiki/Help_centre/Account#sign-in)，或账号被盗。
```

邮箱的 `loss of access` 可以有很多原因，除丢失外也可能有其他

**Comment by CloneWith** [comment]
> announcement message is a new thing, are there any official Chinese name for it yet? i didn't find it and the main page is not translated as well so I just translated the name on my own

这个词翻译的没问题 👍

File: wiki/People/Account_support_team/zh.md (hunk 3 above)
Comment (FlyingFish23) [reply wiki/People/Account_support_team/zh.md:22]: 这个建议太好了omg

File: wiki/People/Account_support_team/zh.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
+# 账号支持团队
+
+*对于管理特定子论坛的团队，请参见[技术支持团队](/wiki/People/Technical_Support_Team)*。
+
+**账号支持团队** 是管理用户账号，并接受禁言或账号限制申诉的一个团队。
+
+## 联系方式
+
+**只能**通过给[accounts@ppy.sh](mailto:accounts@ppy.sh)，[support@ppy.sh](mailto:support@ppy.sh)，[privacy@ppy.sh](mailto:privacy@ppy.sh)或[tournaments@ppy.sh](mailto:tournaments@ppy.sh)发送邮件来联系账号支持团队。为了保护他们不受到太多不需要的关注，团队成员的身份是保密的。
```
Comment (CloneWith) [inline wiki/People/Account_support_team/zh.md:9]:
```suggestion
**只能** 通过给[accounts@ppy.sh](mailto:accounts@ppy.sh)，[support@ppy.sh](mailto:support@ppy.sh)，[privacy@ppy.sh](mailto:privacy@ppy.sh)或[tournaments@ppy.sh](mailto:tournaments@ppy.sh)发送邮件来联系账号支持团队。为了保护他们不受到太多不需要的关注，团队成员的身份是保密的。
```

File: wiki/People/Account_support_team/zh.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+- 关于管理员不当行为的疑虑。
+- 关于管理员破坏[贡献者公约](/wiki/Rules/Contributor_code_of_conduct)的疑虑。
+- [osu!store](https://osu.ppy.sh/store/listing) 订单中遇到的[支付问题](/wiki/Help_centre/Account#supporter)。
+- [帮助论坛](https://osu.ppy.sh/community/forums/5)中没有或者无法解决的问题，例如[连接问题](/wiki/Help_centre/Client#online-features)。
+
+### [privacy@ppy.sh](mailto:privacy@ppy.sh)
+
+- [数据法定权利的行使](/wiki/Legal/Privacy#数据控制员)和隐私疑虑。
+- **永久性的**[账号删除请求](/wiki/Legal/Privacy#您的权利和控制)。
```
Comment (CloneWith) [inline wiki/People/Account_support_team/zh.md:45]:
```suggestion
- **永久性的** [账号删除请求](/wiki/Legal/Privacy#您的权利和控制)。
```

**Review by CloneWith — COMMENTED** [review]
两处空格修改，别的没问题

**Review by CloneWith — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Pull request merged**